recipe/build.patch

```diff
@@ -12,7 +12,7 @@ diff --git a/dvc/build.py b/dvc/build.py
 new file mode 100644
 index 00000000..84c77c84
 --- /dev/null
-+++ b/dvc/build.py
++++ b/dvc/utils/build.py
```

Comment:
CI build is complaining that the to-be-patched file already exists. What is even the use of patching then?

Reply:
Good catch! Indeed, it is based on the pypi package, which is currently using its own build.py.  Thank you! :pray: